dts/bindings/flash_controller/cdns,qspi-nor.yaml

```diff
@@ -0,0 +1,11 @@
+description: Cadence Quad-SPI NOR flash controller
+
+compatible: "cdns,qspi-nor"
+
+include: flash-controller.yaml
+
+properties:
+    clock-frequency:
```

Comment:
What about the other properties that are defined as part of the linux binding:

like `cdns,fifo-depth`, `cdns,fifo-width`, etc..

https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/spi/cdns,qspi-nor.yaml

Reply:
Hi Galak, Im not quite sure about linux driver, is that the FIFO for the QSPI SRAM?
we do have the QSPI_SRAM_FIFO_ENTRY_COUNT

#define CAD_QSPI_SRAM_FIFO_ENTRY_COUNT          (512 / sizeof(uint32_t))

Reply:
ok, we can ignore this for now.